composer.json

```
@@ -0,0 +1,88 @@
+{
+    "name": "pheature-flags/pheature-flags",
```

```suggestion
    "name": "pheature/pheature-flags",
```
Doesn't matters too much because the Monorepo does not exist as a composer package, but is a nice point for package naming cohrence.